slp2mp4/slp-to-mp4.py

```diff
@@ -51,6 +51,19 @@ def is_game_too_short(num_frames, remove_short):
     return num_frames < MIN_GAME_LENGTH and remove_short
 
 
+def get_num_processes(conf):
+    if conf.parallel_games == "recommended":
+        return psutil.cpu_count(logical=False)
+    else:
+        return int(conf.parallel_games)
```

Comment:
Unfortunately we had to add psutil as a dependency because I could find no other way to get the physical cpu cores. multiprocess.cpu_count and os.cpu_count are both logical cores.

Reply:
The recommended setting needs to estimate the number of parallel games that can be run on a CPU. This setting should hopefully prevent people from having to 'guess and check' the number of games that they can run. 

The biggest determining factor in running multiple dolphin instances is the number of CPU cores because it indicates the number of processes that can be run in parallel without the performance penalty of trying to run multiple processes on the same CPU. Logical cores can only simulate this but cannot get the same performance as the same number of physical cores when the physical cores are stressed.

I have an i7-8700k which has 6 physical cores and 12 logical cores. Running 12 games or more at a time is possible but the framerate drops significantly and is bottlenecked by CPU. I can run 7-9 games at a time (depending on resolution and bitrate) without seeing framerate dropping so physical cores are a better benchmark than logical cores. 

Many systems have the same number of physical and logical cores, in which case the recommendation would not matter. But for systems like mine where the difference between logical and physical cores is great, number of physical cores will always be a better predictor.